internal/repository/repository.go

```diff
@@ -143,6 +143,15 @@ func (r *Repository) Config() restic.Config {
 	return r.cfg
 }
 
+// ID returns the repository ID.
+func (r *Repository) ID() (restic.ID, error) {
+	id, err := restic.ParseID(r.cfg.ID)
+	if err != nil {
+		return restic.ID{}, err
+	}
+	return id, nil
```

Comment:
```suggestion
	return restic.ParseID(r.cfg.ID)
```
results in exactly the same behavior.

Reply:
I removed this method entirely, since it's no longer needed with the method you shared in the next comment.